### ISSUE TITLE
Ingest data [3]

### ISSUE THREAD

**Issue by dancingsushii** [amosproj/amos2024ss04-building-information-enhancer#36]
## User story
1. As a Backend Developer, I want to ingest the following data (https://discord.com/channels/1229761892910628934/1229762332603973642/1233370157464682538) to enhance our application with geolocational information.
2. I want / need to extract, transform, and load the data from the CSV to facilitate queries about location information.
3. So that our application can provide comprehensive geospatial information in Germany.

## Acceptance criteria
* Have some notes on data sources
* Create a quick summary file about the data: categories, possible filters, column information.
* Upload it to Google Drive

## Definition of done (DoD)
* Added only after week 5

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
